mandatory/4-tax.js

```diff
@@ -17,8 +21,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  return "£" + calculateSalesTax(price).toFixed(2);
```

Comment:
This function was written correctly but will not pass the test because it uses "calculateSalesTax" which had an error.